### PR TITLE
Use "Secure Signals" instead of "ESP"

### PR DESCRIPTION
**Pull request by alex-yau-ttd**
Rename usages of "ESP" to use "Secure Signals" instead, in terms of documentation, tasks and folder names.

**Manual testing**
Running Launch Secure Signals (Chrome) 
image

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.formatOnSave": true,
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```
Comment (alex-yau-ttd) [inline .vscode/settings.json:14]: Unrelated change, due to VS code upgrade

**Review by lionell-pack-ttd — APPROVED** [review]
